go.mod

```diff
@@ -7,23 +7,23 @@ require (
 	github.com/golang/mock v1.6.0
 	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/google/uuid v1.3.0 // indirect
-	github.com/imdario/mergo v0.3.12 // indirect
+	github.com/imdario/mergo v0.3.13 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-test/v5 v5.0.0
 	github.com/kubernetes-csi/external-snapshotter/client/v6 v6.1.0
-	github.com/miekg/dns v1.1.48 // indirect
-	github.com/prometheus/client_golang v1.13.0
+	github.com/miekg/dns v1.1.50 // indirect
+	github.com/prometheus/client_golang v1.13.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.0
 	google.golang.org/grpc v1.50.1
 	google.golang.org/protobuf v1.28.1
-	k8s.io/api v0.25.2
-	k8s.io/apimachinery v0.25.2
-	k8s.io/apiserver v0.25.2
-	k8s.io/client-go v0.25.2
-	k8s.io/component-base v0.25.2
-	k8s.io/component-helpers v0.25.2
-	k8s.io/csi-translation-lib v0.25.2
+	k8s.io/api v0.25.3
+	k8s.io/apimachinery v0.25.3
+	k8s.io/apiserver v0.25.3
+	k8s.io/client-go v1.5.2
```

Comment:
Is this right? It looks to be a very old version. I'm surprised tests aren't breaking.